### ISSUE TITLE
Config is not consistent with style guide

### ISSUE THREAD

**Issue by makbol** [google/eslint-config-google#36]
Example: [disabled keyword-spacing](https://github.com/google/eslint-config-google/blob/master/index.js#L202) vs. [4.6.2 Horizontal whitespace](https://google.github.io/styleguide/jsguide.html#formatting-horizontal-whitespace)

**Comment by makbol** [comment]
Hello @philipwalton. I wanted to point out there's **disabled** rule [`keyword-spacing`](http://eslint.org/docs/rules/keyword-spacing) which
> enforce consistent spacing before and after keywords

in line [202](https://github.com/google/eslint-config-google/blob/master/index.js#L202) whereas, in [style guide](https://google.github.io/styleguide/jsguide.html#formatting-horizontal-whitespace) we read
>Use of horizontal whitespace depends on location, and falls into three broad categories: leading (at the start of a line), trailing (at the end of a line), and internal. Leading whitespace (i.e., indentation) is addressed elsewhere. Trailing whitespace is forbidden.

>Beyond where required by the language or other style rules, and apart from literals, comments, and JSDoc, a single internal ASCII space also appears in the following places only.
>1. **Separating any reserved word (such as `if`, `for`, or `catch`) from an open parenthesis (`(`) that follows it on that line**
> 2. ...

Therefore I could write like:
```javascript
if(condition){
  // ...
}
```
it won't be pointed by linter but according to style guide it's not correct. I just catch this particular case but I think there might be more.

**Comment by philipwalton** [comment]
Thanks, I actually don't remember now why that rule wasn't set.

On initial read, it seems like it should be set to:

```js
'keyword-spacing': 2, // defaults to {before: true, after: true}
```

Does that look right to you?

**Comment by philipwalton** [comment]
Sorry, I forgot to do this. Fixed via e24704c7df378d35aeaeb49dad1a29ebd06c1d6c. I'll cut a new version shortly.
